src/__tests__/custom-document/custom-document.test.tsx

```diff
@@ -0,0 +1,109 @@
+import React from 'react';
+import '@testing-library/jest-dom';
+import { render, screen } from '@testing-library/react';
+import { getPage } from '../../index';
+import CustomDocumentWithGIP_Page from './__fixtures__/custom-document-with-gip/pages/page';
+import CustomApp from './__fixtures__/custom-document-with-gip/pages/_app';
+
+beforeEach(() => {
+  // Suppress validateDOMNesting error logs
```

Comment:
Should probably document this; -- both the error and the workaround.

Reply:
You're right. I didn't realise final users would get the same issue. :)